src/pages/home/Home.styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+export const StudentsContainer = styled.div`
+  width: 100%;
+  padding-top: 20px;
+  min-height: 100vh;
+`;
+export const FiltersContainer = styled.div`
+  display: flex;
+`;
+interface StatusProps {
+  $isActive: boolean;
+}
+export const ButtonContainer = styled.div``;
+export const StatusText = styled.strong<StatusProps>`
+  background-color: ${(props) => (props.$isActive ? "#20C92B" : "#7C7C7C")};
+  color: white;
+  display: inline-block;
+  width: 100px;
+  text-align: center;
+  padding: 2px 15px;
+  font-size: 20px;
+  border-radius: 10px;
+  font-weight: 700;
+`;
+interface SituationProps {
+  status: string;
+}
+
+export const SituationText = styled.strong<SituationProps>`
+  background-color: ${(props) =>
+    props.status === "Al día" ? "#20C92B" : "#E32626"};
+  color: white;
+  display: inline-block;
+  width: 100px;
+  text-align: center;
+  padding: 2px 15px;
+  font-size: 20px;
+  border-radius: 10px;
+  font-weight: 700;
+`;
```

Comment:
Acá hice las mismas modificaciones que en el anterior:
```suggestion
export const SituationText = styled.p<SituationProps>`
background-color: ${(props) =>
  props.status === "Al día" ? "#20C92B" : "#E32626"};
  color: white;
  display: flex;                 
  align-items: center;            
  justify-content: center;       
  width: 136px;
  height: 32px;
  margin: 0 auto;            
  font-size: 16px;
  border-radius: 10px;
  font-weight: 700;
`;
```

Fijate que acá los estilos de SituationText y StatusText son los mismos, habría que unificar todo en un solo estilo y manejarlo con las props